.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "*.wyam": "csharp"
```

Comment:
ℹ️ FYI this is to ensure that anyone who opens this in VSCode will have the `.wyam` files mapped to C# in the editor for highlighting, etc. (can be a separate PR if you want)